packages/repocop/src/remediations/shared-utilities.test.ts

```diff
@@ -114,3 +115,11 @@ describe('Parsing the tags from an aws_cloudformation_stacks_object', () => {
 		expect(result.guRepoName).toBeUndefined();
 	});
 });
+
+describe('removeRepoOwner', () => {
+	it('should strip "guardian/" from the full repo name', () => {
```

Comment:
```suggestion
	it('should strip the owner from the full repo name', () => {
```
non-blocking, just reflecting the more generic function name